charts/deepgram-self-hosted/CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ### Added
 
+### Changed
```

Comment:
No need for `Changed`, `Added`, or `Fixed` sections under "Unreleased" when we don't have anything there yet!

Reply:
Removed those - left the top-level `Unreleased` section. LMK if you don't even want that remaining!